apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -65,4 +67,4 @@ function LoginPage(props: LoginPageProps) {
   )
 }
 
-export default withRouter(LoginPage)
+export default LoginPage
```

Comment:
`withRouter` is not necessary, this will always be wrapped by the top-level router